src/exercise/04.js

```diff
@@ -1,41 +1,53 @@
 // useState: tic tac toe
 // http://localhost:3000/isolated/exercise/04.js
 
-import * as React from 'react'
+import React, { useState, useEffect } from 'react';
```

Comment:
I wonder if these imports are necessary any longer (since we're abstracting their use in useLocalStorageState)

Reply:
You are right, they were from prev exercise, no longer needed